proc/syscall_windows.go

```diff
@@ -22,8 +22,53 @@ type _THREAD_BASIC_INFORMATION struct {
 	BasePriority   int32
 }
 
+type _CREATE_PROCESS_DEBUG_INFO struct {
+	File                syscall.Handle
+	Process             syscall.Handle
+	Thread              syscall.Handle
+	BaseOfImage         uintptr
+	DebugInfoFileOffset uint16
+	DebugInfoSize       uint16
+	ThreadLocalBase     uintptr
+	StartAddress        uintptr
+	ImageName           *byte
```

Comment:
Any reason this is `*byte` instead of `uintptr`?  In particular, this pointer is in the address space of the debuggee, so can't be derefed as a `*byte` from Go anyway.